### DISCUSSION TITLE
Feature request: swap pages up/down

### DISCUSSION THREAD

**Discussion by paultpearson** [xournalpp/xournalpp#3635] [Ideas] [answered]
Would it be possible to add a feature that would move the current page up or down in the list of pages? For instance, to swap page 3 and page 2, go to page 3 and choose "move page up one page".

**Comment by netlimpopo** [comment, marked as answer]
Press F12, this yields the miniature preview panel on the side.  Look at the bottom of the window. Press the Up-arrow symbol or the down-arrow on the bottom of the window to move the current page 1 position up respectively down.

**Comment by paultpearson** [comment]
Cool! Thank you!
